src/blender_mqo/__init__.py

```diff
@@ -34,7 +34,10 @@ def register_updater(bl_info):
     config.repository = "blender-mqo"
     config.current_addon_path = os.path.dirname(os.path.realpath(__file__))
     config.branches = ["master", "develop"]
-    config.addon_directory = config.current_addon_path[:config.current_addon_path.rfind("/")]
+    if os.name == "nt":
+        config.addon_directory = config.current_addon_path[:config.current_addon_path.rfind("\\")]
+    else:
+        config.addon_directory = config.current_addon_path[:config.current_addon_path.rfind("/")]
```

Comment:
実行OSに応じてパスのセパレータを取得する便利関数があるため、こちらを利用するとよりきれいな形で実装できるかと思います
https://github.com/nutti/blender-mqo/blob/80a732c0dce41465f9a8855961ae8eed19bbd079/src/blender_mqo/utils/addon_updater.py#L14-L17

```suggestion
    config.addon_directory = config.current_addon_path[:config.current_addon_path.rfind(utils.addon_updater.get_separator())]
```